src/css/main.css

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
@gautamvij you still have main.css file in this commit. Remove from cache and filter from branch